packages/backend/src/apps/m365-excel/index.ts

```diff
@@ -0,0 +1,17 @@
+import type { IApp } from '@plumber/types'
+
+import actions from './actions'
+
+const app: IApp = {
+  name: 'M365 Excel',
+  key: 'm365-excel',
+  iconUrl: '{BASE_URL}/apps/m365-excel/assets/favicon.svg',
+  authDocUrl: 'https://guide.plumber.gov.sg/user-guides/actions/m365-excel',
+  baseUrl: 'https://www.office.com',
+  apiBaseUrl: 'https://graph.microsoft.com',
+  supportsConnections: true,
+  primaryColor: '000000',
```

Comment:
a bit out of scope: but i just realised is this ever used cus the color code in every app doesn't even have a `#` added which means the background of the app is always white?

Reply:
I actually have no clue about this.... i feel like i dont think this is being used. We probably can remove this anyway, i don't think we're using it at all